src/components/TravelInfo.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import { TextField, InputLabel, Typography } from "@mui/material";
+import style from "../components/style//TravelInfo.module.css";
+
+function TravelInfo(props: any) {
+  let travelInfoData = {};
+
+  const [form, setForm] = useState({});
+
+  function handleForm(e: React.ChangeEvent<HTMLInputElement>, key: string) {
+    setForm({
+      ...form,
+      [key]: e.target.value,
+    });
+  }
+  travelInfoData = {
+    ...form,
+  };
+
+  props.onSaveTravelInfoData(travelInfoData);
```

Comment:
At what moment is this function called? What's the trigger for it? Is this part of `handleForm` function? 🤔

Reply:
That function has nothing to do with the handleform function, it is called every time the page is rendered, I can insert it into the handleform function so that it is called every time it is called... which way is correct?